cmd/zed/serve/command.go

```diff
@@ -53,6 +53,8 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c.conf.Version = cli.Version
 	c.logflags.SetFlags(f)
 	f.IntVar(&c.brimfd, "brimfd", -1, "pipe read fd passed by brim to signal brim closure")
+	c.conf.CORSAllowedOrigins = []string{"*.observableusercontent.com", "localhost"}
+	f.Var((*cli.CommaStringsFlag)(&c.conf.CORSAllowedOrigins), "cors.origins", "comma-separated list of CORS allowed origins")
```

Comment:
I don't really like doing comma strings, can we rather just have the ability for the user to specify the flag multiple times? I know we did this somewhere...

Reply:
Yes, happy to make this change.

Reply:
Ah yes, this one:

```
    -I source file containing Zed query text (may be used multiple times)
```

Reply:
@mattnibs: Done in 0ecdef640cd450f951233341f7d65168e52e9443.